CHANGELOG.latest.md

```diff
@@ -1,2 +1,9 @@
-- Introduced New AdServices Integration on iOS. Call `EnableAdServicesAttributionTokenCollection()` to enable this integration.
-More information [in our docs](https://docs.revenuecat.com/docs/apple-search-ads) and in the [announcement blogpost](https://www.revenuecat.com/blog/iad-vs-adservices-whats-the-difference/).
+### Important changes
+Starting with this version you need to update to Unity IAP 4.4.0+ if you are using Unity IAP together with RevenueCat in your project. We removed support for older versions of Unity IAP since it upgraded to Android billing client 4.
```

Comment:
Couple of small notes: 
- adding how to update
- clarifying that they can still use older versions if needed
- not going into detail about billing client 4, it's kind of into the weeds for Unity devs

```suggestion

### ⚠️ Important: If you're using RevenueCat along with Unity IAP side-by-side ⚠️
Starting with this version you need to use Unity IAP 4.4.0+ if you are using Unity IAP together with RevenueCat in your project. You can update from Unity Package Manager.

If you need to use an older version of Unity IAP, you can continue to use purchases-unity < 4.2.0. 
```

Reply:
Ahh yeah, that's much better. Thanks for writing it out!